modules/twinklespeedy.js

```diff
@@ -1993,7 +1992,7 @@ Twinkle.speedy.callback.evaluateUser = function twinklespeedyCallbackEvaluateUse
 	if (form.notify.checked) {
 		$.each(normalizeds, function(index, norm) {
 			if (Twinkle.getPref('notifyUserOnSpeedyDeletionNomination').indexOf(norm) !== -1) {
-				if (norm === 'g6' && ['disambig', 'copypaste'].indexOf(values[index]) === -1) {
+				if (norm === 'g6' && 'copypaste'.indexOf(values[index]) === -1) {
```

Comment:
I think `indexOf` in this case is meant to be used on an array. Maybe you meant `'copypaste' === values[index]`? Though I think your code might still work, coincidentally.

Reply:
`'copypaste' !== values[index]` but yeah.  The blocks around it do the same thing, weirdly.

Reply:
@MusikAnimal Done

Reply:
> I think `indexOf` in this case is meant to be used on an array. 

It can also be used on an string, in which case it is treated as an array of its characters. Though obviously that isn't intended here.